typescript/src/auth/utils/jwt.ts

```diff
@@ -190,7 +192,8 @@ export async function generateWalletJwt(options: WalletJwtOptions): Promise<stri
   };
 
   if (Object.keys(options.requestData).length > 0) {
-    claims.req = options.requestData;
+    const sortedData = sortKeys(options.requestData);
+    claims.reqHash = hash(Buffer.from(JSON.stringify(sortedData)));
```

Comment:
Does this play nicely with bigint values? Do we need to include a replacer function for bigint values?

Reply:
good point. solving